core/src/main/java/com/orange/ouds/core/component/OudsTag.kt

```diff
@@ -216,6 +216,22 @@ private fun betweenAssetAndLabelSpace(size: OudsTag.Size): Dp {
     }
 }
 
+@Composable
+private fun backgroundColor(status: OudsTag.Status, hierarchy: OudsTag.Hierarchy): Color {
+    return when (hierarchy) {
+        OudsTag.Hierarchy.Emphasized -> status.color()
+        OudsTag.Hierarchy.Muted -> status.mutedColor()
+    }
+}
+
+@Composable
+private fun assetColor(status: OudsTag.Status, hierarchy: OudsTag.Hierarchy): Color {
```

Comment:
If I am correct, an asset represents the bullet, the icon or the loader. Thus maybe we should rename this one to `iconColor`?

Likewise, there is also an `assetPadding`  line 139 and I don't know if we should rename it to `iconPadding` or not.